README.md

```diff
@@ -2,57 +2,46 @@
 
 An opinionated list of awesome [ownCloud Infinite Scale](https://github.com/owncloud/ocis) apps, extensions, services and other resources.
 
+This repository does not contain any code but references the sourcing repositories. It is therefore just a reference list. Some of the web apps are used via the deployment examples in the [admin docs](https://doc.owncloud.com/ocis/next/depl-examples/ubuntu-compose/ubuntu-compose-prod.html) and have their origin in the [web-extensions](https://github.com/owncloud/web-extensions) repository while others use external repositories and need to be added manually to the web UI - you can use the `web-extensions` repo as example for a how-to.
```

Comment:
web-app-skeleton is the boilerplate and example, not web-extensions.

Reply:
The boilerplate link is below in section `Build your own`.
Referencing `admin docs` and  `web-extensions` is intended to see how we integrated them via `docker compose`. We yesterday have updated the `ocis_full` deployment in the ocis repo with most of the extensions listed.